executor-scripts/linux/ipv6

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+[ -n "$VERBOSE" ] && set -x
+
+yesno() {
+	case "$1" in
+	yes|YES|true|TRUE|1)
+		echo 1
```

Comment:
See `is_int` comment. You can use the `true` and `false` builtins, or return, to generate the return value here.